### PR TITLE
Add mount label man page content

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Dan Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by mrunalp** [comment]
We don't need to pull in the  Godeps/_workspace/src/github.com/opencontainers/runtime-spec/specs-go/config.go change as the MountLabel change is already included under struct Linux.

**Comment by rhatdan** [comment]
Fixed

**Comment by mrunalp** [comment]
Could you move this above `--network`? We recently sorted the options in alphabetic order. Thanks!

**Comment by mrunalp** [comment]
LGTM

File: man/ocitools-generate.1.md [hunk 1]
```diff
@@ -222,7 +234,7 @@ colon:
 
 You can use SELinux to add security to the container.  You must specify the process label to run the init process inside of the container using the --selinux-label.
 
-    # ocitools generate --bind /var/db:/data1  --selinux-label system_u:system_r:svirt_lxc_net_t:s0:c1,c2 --rootfs /var/lib/containers/fedora --args bash
+   # ocitools generate --bind /var/db:/data1  --selinux-label system_u:system_r:svirt_lxc_net_t:s0:c1,c2 --mount-label system_u:object_r:svirt_sandbox_file_t:s0:c1,c2 --rootfs /var/lib/containers/fedora --args bash
```
Comment (wking) [inline man/ocitools-generate.1.md]: Busted rebase partially rolling back #46?

File: man/ocitools-generate.1.md [hunk 2]
```diff
@@ -79,6 +79,18 @@ inside of the container.
 **--mount-cgroups**=[rw|ro|no]
   Mount cgroups.  The default is `no`.
 
+**--mount-label**=[=*MOUNTLABEL*]]
```
Comment (wking) [inline man/ocitools-generate.1.md]: Redundant equal signs?  I'd guess this should be **--mount-label**=[*MOUNTLABEL*].

File: man/ocitools-generate.1.md (hunk 1 above)
Comment (rhatdan) [inline man/ocitools-generate.1.md]: What did I break?

Comment (wking) [inline man/ocitools-generate.1.md]: Markdown literals need four-space indents (or fencing).

Comment (rhatdan) [inline man/ocitools-generate.1.md]: Ok I was looking for something in the text I changed.  I fixed the markdown.

**Pull request merged**